open-machine-learning-jupyter-book/ml-fundamentals/classification/yet-other-classifiers.md

```diff
@@ -14,3 +14,243 @@ kernelspec:
 ---
 
 # Yet other classifiers
+
+In this second classification section, you will explore more ways to classify numeric data. You will also learn about the ramifications for choosing one classifier over the other.
```

Comment:
When I build the book, nothing appears in this yet-other-classifiers.html page.

Reply:
I build in local, it can appear the content of the file "yet-other-classifiers.md"

Reply:
I build in local, the content of the file"yet-other-classifiers.md" can appear.